include/crow/utility.h

```diff
@@ -541,17 +541,20 @@ namespace crow
 
         inline static std::string base64decode(const char* data, size_t size, bool urlsafe = false)
         {
-            // clang-format off
-            std::unordered_map<char, unsigned char> key ({
-            {'A',  0},{'B',  1},{'C',  2},{'D',  3},{'E',  4},{'F',  5},{'G',  6},{'H',  7},{'I',  8},{'J',  9},
-            {'K', 10},{'L', 11},{'M', 12},{'N', 13},{'O', 14},{'P', 15},{'Q', 16},{'R', 17},{'S', 18},{'T', 19},
-            {'U', 20},{'V', 21},{'W', 22},{'X', 23},{'Y', 24},{'Z', 25},{'a', 26},{'b', 27},{'c', 28},{'d', 29},
-            {'e', 30},{'f', 31},{'g', 32},{'h', 33},{'i', 34},{'j', 35},{'k', 36},{'l', 37},{'m', 38},{'n', 39},
-            {'o', 40},{'p', 41},{'q', 42},{'r', 43},{'s', 44},{'t', 45},{'u', 46},{'v', 47},{'w', 48},{'x', 49},
-            {'y', 50},{'z', 51},{'0', 52},{'1', 53},{'2', 54},{'3', 55},{'4', 56},{'5', 57},{'6', 58},{'7', 59},
-            {'8', 60},{'9', 61},{urlsafe ? '-' : '+', 62},{urlsafe ? '_' : '/', 63}});
-
-            // clang-format on
+            // We accept both regular and url encoding here, as there does not seem to be any downside to that.
+            // If we want to distinguish that we should use +/ for non-url and -_ for url. Removing the
+            // urlsafe parameter is a breaking change, thus we simply ignore it for now.
+            static_cast<void>(urlsafe);
```

Comment:
Since there wasn't a version released with this function, a breaking change shouldn't be an issue (with the only exception being [this line](https://github.com/CrowCpp/Crow/blob/8f4830b5a14e5ebc2ae9fe70c7d0a3dbc7f4763a/docs/guides/auth.md?plain=1#L32) needing to be changed).